pkg/agent/agent.go

```diff
@@ -243,6 +266,13 @@ func (i *Initializer) initOpenFlowPipeline() error {
 		return err
 	}
 
+	// On windows platform, hostnetwork flows are needed to be set to ensure host can
+	// communicate with outside.
```

Comment:
Feel the comments are not clear. As the flows are also explained in comments in func initHostNetworkFlows(), maybe simply say: hostNetwork flows are needed for host traffic.